src/figuresClasses.ts

```diff
@@ -1,19 +1,71 @@
 export interface Figure {
-
+  shape: string;
```

Comment:
Consider defining an enum for the shape and a type for the color and using them further in the code